frontend/src/concepts/pipelines/content/createRun/RunForm.tsx

```diff
@@ -38,9 +38,12 @@ const RunForm: React.FC<RunFormProps> = ({ data, onValueChange }) => (
       />
     </FormSection>
     <PipelineSection
-      onLoaded={(loaded) => {
-        onValueChange('pipelinesLoaded', loaded);
-      }}
+      onLoaded={React.useCallback(
```

Comment:
The reason to add useCallback was becasuse there was infinite loop error throwing in the console window

Reply:
I'm reviewing this later, but it's great that you add your own reviews to add more context, this is a good practice and I encourage you all if you do it :)

Reply:
`PipelineSection` should be changed to address this issue. After which there will be no need for `useCallback` to workaround the infinite loop.

```
  React.useEffect(() => {
    onLoaded(loaded);
  }, [onLoaded, loaded]);
```
Should be:
```
  React.useEffect(() => {
    onLoaded(loaded);
  }, [loaded]);
```
This `useEffect` should only run whenever the value of `loaded` changes and not every time the `onLoaded` function itself changes.

Reply:
@christianvogt , I have done the same thing, but my test cases are failing

Reply:
mmmm I wouldn't go with that @christianvogt we are removing a needed dependency from the dependency array, sometimes that's the way to go but I would try to avoid it (that's why we have that lint check right in place).

I'm thinking about the best approach here, we can start a discussion around it.